src/Comparer/StubValueComparer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Diff\Comparer;
+
+/**
+ * @since 3.0
+ *
+ * @license GPL-2.0+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class StubValueComparer implements ValueComparer {
```

Comment:
I don't think there is much value in implementing a class that is named "stub" and behaves in a way that can turn out to be so confusing that I really hope nobody is ever using this in production. Instead you can simply use the existing StrictComparer in the test you wrote this stub class for. Or do I miss an intended use case for this stub class?